homeassistant/components/ads/cover.py

```diff
@@ -105,7 +105,12 @@ def __init__(
         self._ads_var_open = ads_var_open
         self._ads_var_close = ads_var_close
         self._ads_var_stop = ads_var_stop
-        self._device_class = device_class
+        self._attr_device_class = device_class
+        self._attr_supported_features = SUPPORT_OPEN | SUPPORT_CLOSE
+        if self._ads_var_stop is not None:
```

Comment:
```suggestion
        if ads_var_stop is not None:
```
Not mandatory, easier to read